src/acachecontrol/cache.py

```diff
@@ -81,3 +97,31 @@ def _dict_hash(input_dict: Dict) -> str:
         return hashlib.sha256(
             json.dumps(sorted(input_dict.items())).encode()
         ).hexdigest()
+
+    @staticmethod
+    def parse_cache_control_header(headers) -> Dict:
+        """Parse cache-control header, get max-age value.
+
+        Args:
+            headers: any dict-like object
+
+        Returns:
+            dict
```

Comment:
I think it's better to add a description of the returned value and an example.

Reply:
added